providers/common/ciphers/cipher_aes.c

```diff
@@ -12,6 +12,8 @@
 #include "cipher_aes.h"
 #include "internal/provider_algs.h"
 
+#define AES_FLAGS (EVP_CIPH_FLAG_DEFAULT_ASN1)
```

Comment:
This change no longer seems necessary. Similar comment for the other provider cipher files.

Reply:
Yup

Reply:
I've also removed that flag among provided implementations that already have it.

Reply:
Ok. Are there any that *don't* have it that now need EVP_CIPH_FLAG_CUSTOM_ASN1?

Reply:
Nope.  It will be all up to @slontis to add that flag for his RC2 port in #9991